releases/release-6.1.3.md

```diff
@@ -0,0 +1,90 @@
+---
+title: TiDB 6.1.3 Release Notes
+---
+
+# TiDB 6.1.3 Release Notes
+
+Release date: xx, xx, 2022
+
+TiDB version: 6.1.3
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.1/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v6.1.3#version-list)
+
+## Improvements
+
```

Comment:
- PD
   - Optimize the lock granularity and improve the processing ability of heartbeat under high concurrency [#5635](https://github.com/tikv/pd/pull/5635) @[rleungx](https://github.com/rleungx)